ntbs-service/Models/Entities/Specimen.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using ntbs_service.Helpers;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace ntbs_service.Models.Entities
+{
+    [NotMapped]
+    public class Specimen
+    {
+        public int NotificationId { get; set; }
+
+        [Display(Name = "Lab number")]
+        public string ReferenceLaboratoryNumber { get; set; }
+
+        [Display(Name = "Speciment Type")]
+        public string SpecimenTypeCode { get; set; }
+
+        [Display(Name = "Specimen Date")]
+        public DateTime? SpecimenDate { get; set; }
+        public string LaboratoryName { get; set; }

```

Comment:
Missed Display attribute

Reply:
I don't need those properties based on this:
https://airelogic-nis.atlassian.net/wiki/spaces/R2/pages/180125711/View+matched+specimen